feature/gribi/otg_tests/mpls_decap/README.md

```diff
@@ -0,0 +1,118 @@
+# PF-1.7 MPLS Decapsulation
+
+Create a policy-forwarding configuration using gNMI to decapsulate MPLS
+in GRE and UDP packets which are sent to a loopback address and apply to
+the DUT.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+TODO: Complete test environment setup steps
+
+inner_ipv6_dst_A = "2001:aa:bb::1/128"
+inner_ipv6_dst_B = "2001:aa:bb::2/128"
+inner_ipv6_default = "::/0"
+
+ipv4_inner_dst_A = "10.5.1.1/32"
+ipv4_inner_dst_B = "10.5.1.2/32"
+ipv4_inner_default = "0.0.0.0/0"
+
+outer_ipv6_src =      "2001:f:a:1::0"
+outer_ipv6_dst_A =    "2001:f:c:e::1"
+outer_ipv6_dst_B =    "2001:f:c:e::2"
+outer_ipv6_dst_def =  "2001:1:1:1::0"
+outer_dst_udp_port =  "6635"
+outer_dscp =          "26"
+outer_ip-ttl =        "64"
+
+## Procedure
+
+### TE-18.1.3 - MPLS in GRE decapsulation set by gNMI
+
+Canonical OpenConfig for policy forwarding, matching IP prefix with action
+decapsulate in GRE. # TODO: Move to dedicated README
+
+```yaml
+openconfig-network-instance:
+  network-instances:
+    - network-instance: "DEFAULT"
+      afts:
+        policy-forwarding:
+          policies:
+            policy: "default decap rule"
+              config:
+                policy-id: "default decap rule"
+                type: PBR_POLICY
+              rules:
+                rule: 1
+                  config:
+                    sequence-id: 1
+                  ipv6:
+                    config:
+                      destination-address: "decap_loopback_ipv6"
+                  action:
+                    decapsulate-mpls-in-gre: TRUE             # TODO: add to OC model/PR in progress
+```
```

Comment:
Add the procedure steps here for PF-1.7.1 . (they are similar to PF-1.7.2, but for GRE)